uint8[] robotID
uint64[] objectID
float32[] azimuth
float32[] elevation
float32[] radius
float32[] confidence
float64[] timestamp
rosMsgs/s_camera_type[] cameraType
float32[] cameraX
float32[] cameraY
float32[] cameraZ
float32[] cameraPhi
---
